### PR TITLE
Migrated to NET 6 (And made use of the file-scoped namespaces feature).

### LINKED ISSUE #10: Combine Layers for Skia
An SerializeableDocument extension for combining all layers to a SKBitmap in the PixiParser.Skia project would be great.

It has already been implemented in PixiBot [here](https://github.com/PixiEditor/PixiBot/blob/main/src/PixiBot.Bot/Helpers/Extensions/DocumentHelpers.cs#L8-L42) so it should be just a matter of a copy pasta

### PR DESCRIPTION
**Pull request by StarlkYT**
Resolve #10
Add `.editorconfig`

The tests run fine, though I haven't tested it with PixiEditor (I don't see why it'd fail anyway.)

### CODE REVIEW

File: src/PixiParser.Benchmarks/Program.cs [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-    class Program
+    static void Main()
     {
-        static void Main()
-        {
-            BenchmarkRunner.Run<Benchmarks>();
-        }
+        BenchmarkRunner.Run<Benchmarks>();
     }
 }
```
Comment (CPKreuz) [inline src/PixiParser.Benchmarks/Program.cs:11]: I think you could just make this a top-level statement program file

File: src/PixiParser.Skia/SkiaExtensions.cs [hunk 2]
```diff
@@ -1,58 +1,90 @@
...
+        using SKData data = image.Encode();
+
+        layer.PngBytes = data.AsSpan().ToArray();
+        layer.Width = image.Width;
+        layer.Height = image.Height;
+
+        return layer;
+    }
+
+    public static SKBitmap ToSKBitmap(this SerializableDocument document)
```
Comment (CPKreuz) [inline src/PixiParser.Skia/SkiaExtensions.cs:55]: Can you add some documentation text, something along the lines
```
Draws all layers on top of a SKBitmap
```
should do


And actually rename the method to `LayersToSKBitmap`

**Review by CPKreuz — CHANGES_REQUESTED** [review]
It would be great if you could put your changes into multiple commits next time so it's easier to read the diff

File: src/PixiParser.Benchmarks/Program.cs (hunk 1 above)
Comment (StarlkYT) [reply src/PixiParser.Benchmarks/Program.cs:11]: Indeed! I totally forgot that 🤔

**Comment by StarlkYT** [comment]
I edited it, is it good now?

**Review by CPKreuz — CHANGES_REQUESTED** [review]
There are actually a few warnings regarding

`Use expression body for methods	PixiParser (net6.0), PixiParser`

which come from the editorconfig


Otherwise this seems ready to merge, great work!

**Review by CPKreuz — APPROVED** [review]

**Pull request merged**